internal/config/config.go

```diff
@@ -130,7 +130,7 @@ type AWSConfig struct {
 	IAMProfileWorkerNodes string `yaml:"iamProfileWorkerNodes" validate:"required"`
 	// description: |
 	//   Deploy Persistent Disk CSI driver with on-node encryption. For details see: https://docs.edgeless.systems/constellation/architecture/encrypted-storage
-	DeployCSIDriver *bool `yaml:"deployCSIDriver" validate:"required"`
+	DeployCSIDriver *bool `yaml:"deployCSIDriver"`
```

Comment:
```suggestion
	DeployCSIDriver *bool `yaml:"deployCSIDriver"` # TODO after v2.9 release: re-enable "required" validation
```
Maybe add a TODO here to re-enable the validation